src/azure/storage/signer.rs

```
@@ -67,14 +67,17 @@ impl Signer {
                 ctx.query_append(token);
                 return Ok(ctx);
             }
-            Credential::BearerToken(token) => {
-                ctx.headers.insert(AUTHORIZATION, {
-                    let mut value: HeaderValue = format!("Bearer {}", token).parse()?;
-                    value.set_sensitive(true);
-                    value
-                });
-                return Ok(ctx);
-            }
+            Credential::BearerToken(token) => match method {
+                SigningMethod::Query(_) => {}
```

We should return an error instead.